src/components/PaginationFooter/Dropdowns/DropDown.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import type { SelectChangeEvent } from '@mui/material/Select';
+import { usePaginationContext } from '../context/PaginationProvider';
+import { StyledSelect, StyledTypography } from '../styles';
+
+interface Props {
+  name: string;
+  value: number;
+  options: JSX.Element;
+  handleChange: (event: SelectChangeEvent<number>) => void;
+}
+
+const DropDown = ({ name, value, options, handleChange }: Props) => {
+  const { layout, footerStyle, translator, tabIndex } = usePaginationContext();
+  const chartId = layout.qInfo.qId as string;
+  const label = translator.get(`NebulaTableUtils.Pagination.${name}`);
+  const id = `${name}-dropdown`;
```

Comment:
I forgot that the id for the dropdown itself needs to be unique as well, so concat the chartId here